index.js

```diff
@@ -980,6 +980,54 @@
   //. ```
   S.init = S.slice(0, -1);
 
+  //# take :: Number -> [a] | String -> Maybe [a] | Maybe String
```

Comment:
Yeah, I was thinking that but wasn't sure whether the plan was to go down that route. Should we mention it anywhere in the docs that that is how we treat strings?